Python/Scripts/Any/row_level_security_basic.py

```
@@ -0,0 +1,316 @@
+            status_message = status_message.replace("__TEAM_COLUMN__", team_column)
+            status_message = status_message.replace("__TEAM_VALUE__", team_id)
+
+            liquibase_status.message = status_message
+            liquibase_logger.warning(f"Row-level security violation: {error_message}")
+            liquibase_logger.info(f"SQL: {raw_statement[:200]}...")  # Log first 200 chars
+            sys.exit(1)
+
+###
+### Default return code
+###
+False
```

Ruff (0.14.14)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)